snuba/stateful_consumer/control_protocol.py

```diff
@@ -148,6 +148,17 @@ def from_json(cls, json: Mapping[str, Any]) -> ControlMessage:
             )
         )
 
+    def serialize(self) -> Mapping[str, Any]:
```

Comment:
Like noted in getsentry/cdc#22, this isn't really serialization. Renaming this to `to_dict` like it was there probably make sense.